utils/generic-components.stanza

```diff
@@ -654,22 +653,34 @@ public pcb-module micro-usb-connector :
 ; USB2 device connection via USBC connector
 public pcb-module usb2-on-a-usb-c-connector :
   port usb-2 : usb-2
+  port usb-c : usb-c
   public inst con : ocdb/belfuse/SS-52400-003/component
 
   net VBUS (usb-2.vbus.vdd con.vbus)
   net (usb-2.vbus.gnd con.GND)
+  net (usb-c.vbus usb-2.vbus)
 
   net D- (usb-2.data.N con.DN1, con.DN2)
   net D+ (usb-2.data.P con.DP1, con.DP2)
-
+  net (usb-c.data[1] usb-2.data)
   ; RC filter connection for chassis gnd
   res-strap(con.SHIELD, con.GND, 100.0e3)
   cap-strap(con.SHIELD, con.GND, 0.1e-6)
-
+  net (con.SBU1 usb-c.sbu[1])
```

Comment:
These connections would be better moved to the belfuse definition, to be consistent with the hroparts connector. Create the local module in that package to map to usb-c, and then either connector could be using in this reusable module.

Reply:
fixed